apps/web/src/app/admin-console/organizations/core/services/group/group-api.service.ts

```diff
@@ -44,12 +46,28 @@ export class GroupService {
 
     const listResponse = new ListResponse(r, GroupDetailsResponse);
 
-    return Promise.all(listResponse.data?.map((gr) => GroupView.fromResponse(gr))) ?? [];
+    return (await Promise.all(listResponse.data?.map((gr) => GroupView.fromResponse(gr)))) ?? [];
```

Comment:
This `map` callback does not return a `Promise`, so there's no need to `await Promise.all`.

Also, `listResponse.data` is never `null`: [ref](https://github.com/bitwarden/clients/blob/df7e0fca2bd04096215dbcea99e164b4cbbbaff1/libs/common/src/models/response/list.response.ts#L10).

```suggestion
    return listResponse.data.map((gr) => GroupView.fromResponse(gr));
```